cli/src/config/revsets.toml

```diff
@@ -18,3 +18,5 @@ latest(
 '''
 
 'immutable_heads()' = 'trunk() | tags()'
+'immutable()' = '::immutable_heads()'
```

Comment:
This should be `::(immutable_heads() | root())` since that's [the definition that's used internally](https://github.com/martinvonz/jj/blob/c34f35fffe1b8e5932ba27ac4da8ba5a97d417a5/cli/src/revset_util.rs#L189)